modules/Makefile.circleci

```diff
@@ -71,3 +71,17 @@ circle\:deploy-kubernetes:
   fi
 	@sleep 3
 	@$(SELF) kubernetes:list-deployments kubernetes:list-rs kubernetes:list-pods
+
+## Run job on kubernetes after obtaining an exclusive lock
+circle\:run-job-kubernetes:
+	$(call assert,IMAGE_TAG)
+	$(call assert,JOB_KUBERNETES_APP)
+	$(call assert,CIRCLE_BRANCH)
+	@if [ -z "$(CIRCLE_TOKEN)" ]; then \
+	  echo -e "$(call red,WARN:) Running $(JOB_KUBERNETES_APP):$(IMAGE_TAG) without obtaining lock; CIRCLE_TOKEN not defined"; \
```

Comment:
I wanted to retain the relationship to the KUBERNETES_APP var it controls for jobs.